src/pages/overlay/components/overlay/ambassadors/Ambassadors.tsx

```diff
@@ -84,58 +92,74 @@ export default function Ambassadors(props: OverlayOptionProps) {
+        className,
+      )}
+    >
+      <div className="relative z-10 flex flex-col items-center">
         <div
           ref={ambassadorList}
-          className={styles.ambassadors}
+          className="list-fade scrollbar-none -my-[var(--twitch-vertical-padding)] flex w-40 flex-col items-center gap-4 overflow-scroll px-4 py-[calc(var(--twitch-vertical-padding)+var(--list-fade-padding))]"
           onScroll={handleArrowVisibility}
         >
-          {ambassadors.map(([key, ambassador]) => (
+          {ambassadors.map(([key]) => (
             <AmbassadorButton
               key={key}
-              ambassadorKey={key}
-              ambassador={ambassador}
+              ambassador={key}
               onClick={() => {
                 setActiveAmbassador((prev) =>
                   prev.key === key ? {} : { key },
                 );
               }}
               className={classes(
-                styles.ambassadorButton,
-                activeAmbassador.key === key && styles.highlighted,
+                activeAmbassador.key === key && "outline-highlight outline",
               )}
             />
           ))}
         </div>
 
         <button
           ref={upArrowRef}
-          className={classes(styles.arrow, styles.up, styles.hidden)}
+          className={classes(
+            arrowClass,
+            "-top-[var(--twitch-vertical-padding)]",
+            hiddenClass,
+          )}
           onClick={(e) => ambassadorListScroll(e, 250)}
           title="Scroll up"
+          type="button"
+          data-transparent-clicks
         >
-          <IconChevron />
+          <IconChevron className={classes(arrowSvgClass, arrowPathClass)} />
         </button>
 
         <button
           ref={downArrowRef}
-          className={classes(styles.arrow, styles.down)}
+          className={classes(
+            arrowClass,
+            "-bottom-[var(--twitch-vertical-padding)] rotate-180",
+          )}
           onClick={(e) => ambassadorListScroll(e, -250)}
           title="Scroll down"
+          type="button"
+          data-transparent-clicks
         >
-          <IconChevron />
+          <IconChevron className={classes(arrowSvgClass, arrowPathClass)} />
         </button>
       </div>
 
-      {activeAmbassador.key && (
-        <AmbassadorCard
-          key={activeAmbassador.key}
-          ambassador={activeAmbassador.key}
-          onClose={() => setActiveAmbassador({})}
-          className={styles.ambassadorCard}
-        />
-      )}
+      {ambassadors.map(([key]) => (
+        <Transition show={activeAmbassador.key === key} key={key}>
+          <AmbassadorCard
+            key={key}
+            ambassador={key}
+            onClose={() => setActiveAmbassador({})}
+            className="z-0 col-start-2 row-start-1 origin-[center_left] self-center transition-[opacity,transform] data-[closed]:-translate-x-10 data-[closed]:opacity-0 data-[closed]:motion-reduce:translate-x-0"
```

Comment:
Chrome and Firefox are a bit jumpy without `will-change` for me shifting the text and card height a tiny bid after the transition. I think this fixed that.

Also I'd probably up the transition duration a bid. Feels a lot smoother to me.

```suggestion
            className="z-0 col-start-2 row-start-1 origin-[center_left] self-center transition-[opacity,transform] data-[closed]:-translate-x-10 data-[closed]:opacity-0 data-[closed]:motion-reduce:translate-x-0 duration-500 will-change-transform"
```

Reply:
Maybe not 500ms. I have not checked what the other durations are set to.

Reply:
Ah yeh, the old transitions globally were 300ms, Tailwind will default to 150ms. Will bump that back up.